db/model_test.go

```diff
@@ -0,0 +1,73 @@
+// Package db - model test
+// The following file contains all the models for testing.
+
+package db
+
+import (
+	"context"
+
+	"github.com/packethost/pkg/log"
+)
+
+type hardwarePayload struct {
```

Comment:
there are no tests in this file, seems like its better to have it in common_test.go

Reply:
Moved